onnxruntime/test/providers/vitisai/vitisai_basic_test.cc

```diff
@@ -0,0 +1,316 @@
+#pragma comment(lib, "d3d12.lib")
+
+#include "core/providers/vitisai/imp/tensor_proto.h"
+
+#include <wil/result.h>
+#include <gtest/gtest.h>
+#include <cstddef>
+
+#include <iostream>
+#include <string>
+#include <fstream>
+
+using namespace vaip;
```

Comment:
Please do not do this. Do not put everything from the `vaip` namespace into the global namespace. Please use `vaip::foo()` instead of `foo`.